spec/controllers/users_controller_spec.rb

```diff
@@ -27,6 +27,34 @@
     end
   end
 
+  describe '#bids_placed' do
+    context 'when logged in' do
+      it 'should assign auctions that current user have bidded on, presented' do
+        current_bidder = create(:user, sam_status: :sam_accepted)
+        auction = create(:auction)
+        bid = create(:bid, bidder: current_bidder, auction: auction)
+        expect(bid).to be_valid
+        get :bids_placed, { }, user_id: current_bidder.id
```

Comment:
I think it would be helpful here to have newlines between setup, exercise, and expectations (like you did in spec/view_models/user_auction_view_model_spec.rb)